modules/component/src/componentNext.ts

```diff
@@ -98,13 +86,38 @@ export class ComponentNext<P1 extends ComponentProps> {
     )
   }
 
+  /**
+   * Create new component with initial state as undefined
+   *
+   * ```typescript
+   * import {ComponentNext} from '@action-land/component'
+   *
+   * const component = ComponentNext.empty
+   * ```
+   */
   static get empty(): ComponentNext<{
     iState: undefined
     oView: void
   }> {
     return ComponentNext.lift(undefined)
   }
 
+  /**
+   * Add transformation of component's state for a given action
+   *
+   * ```typescript
+   * import {ComponentNext} from '@action-land/component'
+   *
+   * const component = ComponentNext.lift({count: 100})
+   *  .matchR('add', (value: number, state) => ({count: state.count + value}))
+   *  // Adds behaviour to handle action of type Action<number, 'add'>
+   * ```
+   * @typeparam T Action type being handled by operator
+   * @typeparam V Value of action being handled by operator
```

Comment:
- should be `to be handled` (future tense) instead of `being handled` (past tense) for the two lines above
  - the behaviour is being added; `being handled` implies it already exists.
- also remove `by operator`
  - if anything, it should be `by component`